Gruntfile.js

```diff
@@ -18,6 +26,21 @@ module.exports = function(grunt) {
         stripBanners: true,
         banner: '/* <%= pkg.name %> <%= grunt.template.today("yyyy-mm-dd") %> */',
       },
+      static_mappings: {
```

Comment:
I like this approach to a cleaner builds directory

Reply:
yeah, it really bugged me to be using stuff outside of a builds or dist directory.  Ticket #30 captures running it entirely from a builds or dist directory.